### PR TITLE
Fix Unicode-objects must be encoded before hashing

### PR DESCRIPTION
**Pull request by trungdq88**
My python3 build is currently failing because of this. https://travis-ci.com/github/zendesk/zopim-pimp/jobs/353463464

### CODE REVIEW

**Review by blakegong — APPROVED** [review]

**Review by ningt — APPROVED** [review]
Should squash the commits

**Review by yihongang — APPROVED** [review]

File: banditParser.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 def hash(toHash):
 	hash_object = hashlib.sha256()
-	hash_object.update(toHash)
+	hash_object.update(toHash.encode('utf8'))
```
Comment (sfdye) [inline banditParser.py:10]: But this might fail for Python 2, should only encode `toHash` if it's not bytes already

Comment (trungdq88) [reply banditParser.py:10]: Hm... I remember testing this on both py2 and py3 and there was no problem. Could you explain how would this fail for py2?

Comment (sfdye) [reply banditParser.py:10]: Just being paranoid 😄 
BTW now you can completely remove bandit (approved by security) because we are moving to Snyk

**Pull request merged**